scripts/changelog_check.rb

```diff
@@ -71,6 +71,10 @@ def build_structured_git_log(git_log)
   end
 end
 
+def current_branch
+  Open3.capture2('git', 'branch', '--show-current')[0].chomp
```

Comment:
I'd go with the multiline approach but otherwise same same
```suggestion
  output, _status = Open3.capture2('git', 'branch', '--show-current')
  output.chomp
```

Reply:
Resolved indirectly through df0480af974938554d307b0dd4ca5f138d1604af (see https://github.com/18F/identity-idp/pull/6468#discussion_r891359145)